src/components/accessible-autocomplete/accessible-autocomplete.js

```diff
@@ -6,20 +6,34 @@ function AccessibleAutoComplete($module, window, document) {
 
 AccessibleAutoComplete.prototype.init = function init() {
   if (this.$module) {
-    const showAllValues = (this.$module.getAttribute('data-show-all-values') === 'true');
-    const autoselect = (this.$module.getAttribute('data-auto-select') === 'true');
-    const defaultValue = this.$module.getAttribute('data-default-value');
-    const minLength = this.$module.getAttribute('data-min-length');
+    const selectElement = this.$module;
+    const selectOptions = Array.from(selectElement.options);
+    const autocompleteId = selectElement.id;
+    const showAllValues = (selectElement.getAttribute('data-show-all-values') === 'true');
+    const autoselect = (selectElement.getAttribute('data-auto-select') === 'true');
+    const defaultValue = selectElement.getAttribute('data-default-value');
+    const minLength = selectElement.getAttribute('data-min-length');
 
     const configurationOptions = {
-      selectElement: this.$module,
+      selectElement,
       showAllValues,
       autoselect,
       defaultValue,
       minLength,
+      onConfirm: (chosenOption) => {
+        selectElement.value = '';
+        const chosenOptionOrCurrentValue = (typeof chosenOption !== 'undefined')
+          ? chosenOption
+          : document.getElementById(autocompleteId)?.value;
```

Comment:
just to double check, I looked at the compiled output and `?.` is not passed through, so won't be any compatibility issues by using it here